src/server/cmd/worker/main.go

```diff
@@ -71,7 +71,7 @@ func do(ctx context.Context, config interface{}) error {
 	debugclient.RegisterDebugServer(server.Server, debugserver.NewDebugServer(env, env.Config().PodName, pachClient, env.GetDBClient()))
 
 	// Put our IP address into etcd, so pachd can discover us
-	workerRcName := ppsutil.PipelineRcName(pipelineInfo.Pipeline.Name, pipelineInfo.Version)
+	workerRcName := ppsutil.PipelineRcName(pipelineInfo.Pipeline.Project.GetName(), pipelineInfo.Pipeline.Name, pipelineInfo.Version)
```

Comment:
I use `.GetName()` because until this work is done it is not wrong for a repo or a pipeline to have a nil project. Once the work is done, CORE-1046 is a story to change `.GetName()` to `.Name` throughout.

> I think that actually calling the field has the benefit of exposing null pointer issues.

Better than exposing a null pointer by crashing would be to avoid the possibility entirely by making invalid states unrepresentable, but that’s another story for another epic☺